mobile/test/reducers/settings.spec.js

```diff
@@ -14,6 +14,11 @@ describe('settings reducers', () => {
     .toEqual({serverUrl: url, error: null, authorized: false})
   })
 
+  it('should handle SET_ANALYTIC', () => {
+    expect(reducer({analytic: false}, {type: SET_ANALYTIC, analytic: true})).toEqual({analytic: true})
+    expect(reducer({analytic: true}, {type: SET_ANALYTIC, analytic: false})).toEqual({analytic: false})
+  })
```

Comment:
We'll refactor the whole `reducers` spec to test all-in-one: actions, action creators, reducers (cf. https://github.com/cozy/cozy-files-v3/commit/04c18b6b9522080fef6de0454fee03d7dca57bfa)